src/components/InfoPanel.component.js

```diff
@@ -27,19 +27,21 @@ class InfoPanel extends Component {
       activePOIs,
       isStorySelected,
       selectedPOIIndex,
+      previewedPOIIndex,
       isFirstInStory,
       isLastInStory,
       selectedPOI,
+      previewedPOI,
       isEditing,
       removePOIFormMedia,
       setNextPOIInStory,
       setPreviousPOIInStory,
       location
     } = this.props
-
     const isRealTimePOI = location.pathname === ROUTES.FORM
 
-    if (!selectedPOI) {
+    let activePOI = null
```

Comment:
you can just do
```jsx
const activePOI = previewedPOI || selectedPOI
```
this lets you remove lines 59/60 and removes the need for `let` so you can use `const`

Reply:
its also hella elegant and covers so much ground with such little and easy to understand code *sniff*